Makefile

```diff
@@ -75,7 +75,7 @@ integration-unix: build
 
 .PHONY: integration-linux
 integration-linux: build
-	$(info INFO: Starting build $@)
```

Comment:
Is this a mistake? 🤔

Reply:
ah yeah, probably through my hours of head banging the DNS issues. 

Kinda want to redo the integration test to perform the ssh and docker test as separate jobs, so this kinda got put on the back burner.

Reply:
Also expect a GitHub action example..